modules/3d-tiles/src/parsers/parse-3d-tile-point-cloud.js

```diff
@@ -47,13 +48,17 @@ function extractPointCloud(tile) {
   tile.isTranslucent = false;
   tile.isRGB565 = false;
   tile.isOctEncoded16P = false;
+  tile.featureTable = featureTable;
```

Comment:
Just background: The reason I was passing in the feature table was that I wasn't sure that I wanted to store the feature table, since it is not pure data.